### PR TITLE
updates folders and pages to match current IA

### PR DESCRIPTION
**Pull request by dawnkelly09**
This PR should now reflect the correct folders and pages to match the current IA.

### CODE REVIEW

File: develop/blockchains/custom-blockchains/.pages [hunk 1]
```diff
@@ -1,8 +1,9 @@
 title: Custom Blockchains
 nav:
   - index.md
-  - 'Overview': overview.md
+  - 'Overview (FRAME)': frame-overview.md
   - 'Add Existing Pallets': add-existing-pallets.md
   - 'Make a Custom Pallet': make-custom-pallet.md
+  - 'Add Smart Contract Functionality': add-smart-contract.md
```
Comment (eshaben) [inline develop/blockchains/custom-blockchains/.pages:7]:
```suggestion
```

Delete for now

File: develop/blockchains/custom-blockchains/add-smart-contract.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (eshaben) [inline develop/blockchains/custom-blockchains/add-smart-contract.md:1]: Delete

**Review by eshaben — COMMENTED** [review]
Awesome! Thanks for tackling this and sorting it all out. Left comments on the empty files we should get rid of now

**Comment by eshaben** [comment]
I went through the IA and deleted any pages where we don't have content. We can add these as we go, but I don't want to have to go back and delete pages again at the end of the week. I left a select few empty pages because we NEED to get those done

**Review by nhussein11 — APPROVED** [review]
LGTM!

**Pull request merged**